src/mode/mode_handler.ts

```diff
@@ -29,15 +29,18 @@ export default class ModeHandler {
         return currentMode;
     }
 
-    public SetCurrentModeByName(modeName : ModeName) {
+    SetCurrentModeByName(modeName : ModeName) {
         this.modes.forEach(mode => {
             mode.IsActive = (mode.Name === modeName);
         });
 
-        this.setupStatusBarItem(ModeName[modeName].toUpperCase());
+        // Vim never shows -- NORMAL -- in the status bar.
```

Comment:
I put it there intentionally just to make it more obvious, but we can remove it.